unit_test/test_spdm_responder/measurements.c

```diff
@@ -1618,6 +1618,16 @@ void libspdm_test_responder_measurements_case27(void **state)
     void *data;
     size_t data_size;
     size_t measurment_sig_size;
+    uint8_t content_changed;
+    uint8_t measurements_count;
+    uint8_t *measurements;
+    size_t measurements_size;
```

Comment:
Change `measurements_count` to `NumberOfBlocks`,
Change `*measurements` to `MeasurementRecordData`,
Change `measurements_size` to `MeasurementRecordLength`,
then it will be more clear meaning and align with spec.